### PR TITLE
Atmosia Asteroid Space Ruin

### PR DESCRIPTION
**Pull request by ZAMODA**
## About The Pull Request
This PR adds a new space ruin, the atmos asteroid!
It contains a small atmos setup with some engineering tools, as well as some RPD upgrades.
image
### Mapping March
Ckey to receive rewards: Z Man Wit Z Plan

## Why It's Good For The Game
Space deserves some more ruins.
## Changelog
:cl:
add: Added a new space ruin, the Atmos Asteroid!
/:cl:

### CODE REVIEW

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -356,3 +356,9 @@
 	suffix = "the_faceoff.dmm"
 	name = "The Faceoff"
 	description = "What do you get when a meeting of the enemy corporations get crashed?"
+
+/datum/map_template/ruin/space/atmosastroidruin
+	id = "atmosastroidruin"
+	suffix = "atmosastroidruin.dmm"
+	name = "Atmos Astroid"
```
Comment (Fikou) [inline code/datums/ruins/space.dm:363]: it's "Asteroid"

File: code/datums/ruins/space.dm [hunk 2]
```diff
@@ -356,3 +356,9 @@
...
+	description = "A dead atmos tech in a continuously presurizing ruin"
```
Comment (Fikou) [inline code/datums/ruins/space.dm:364]:
```suggestion
	description = "A dead atmos tech in a continuously pressurizing ruin."
```

**Comment by ZAMODA** [comment]
Spelling should be fixed!

**Comment by san7890** [comment]
This must be added to the configuration, see https://github.com/tgstation/tgstation/blob/master/.github/guides/MAPS_AND_AWAY_MISSIONS.md#configuration

**Comment by ZAMODA** [comment]
> This must be added to the configuration, see https://github.com/tgstation/tgstation/blob/master/.github/guides/MAPS_AND_AWAY_MISSIONS.md#configuration

Done

**Comment by ZAMODA** [comment]
Since I got all the issues above fixed I'll wait for any more feedback and if I don't get anything I'll un-draft this PR and set it to open again.

**Comment by ZAMODA** [comment]
Oh God why is GitHub being dumb

**Comment by ZAMODA** [comment]
> Looks good now, but out of curiosity....
> 
> Since the lore of it is that they suffocated themselves, have you considered making it so that the interior of the ruin spawns highly pressurized / filled with CO2?

If i knew how to change the atmos of turfs/add gases directly then i could, but at the moment the best I could do was the broken CO2 miner

**Comment by ZAMODA** [comment]
> > Looks good now, but out of curiosity....
> > Since the lore of it is that they suffocated themselves, have you considered making it so that the interior of the ruin spawns highly pressurized / filled with CO2?
> 
> If i knew how to change the atmos of turfs/add gases directly then i could, but at the moment the best I could do was the exposed CO2 miner

wait no i might have figured it out, give me a minute

**Review by MMMiracles — CHANGES_REQUESTED** [review]
I'm personally a bit iffy on the existence of every major gas miner sitting all in the same ruin since they aren't used that often in the first place due to balance concerns. If some weird meta pops up around the ruin this will probably need revisited but for now it probably wouldn't hurt to see what happens.

As for the ruin itself, mind adding something like a cyclelink_helper to that airlock so it can cycle and at least prevent large amounts of air leakage? 
image
'/obj/effect/mapping_helpers/airlock/cyclelink_helper' is the path for it, the arrows just need to face the airlock you want it to link to for proper cycling. Add that and it should be fine enough for a merge.

**Comment by ZAMODA** [comment]
> As for the ruin itself, mind adding something like a cyclelink_helper to that airlock so it can cycle and at least prevent large amounts of air leakage?

Done! Hopefully that works!

**Pull request merged**